modules/cloud-router-connection/main.tf

```diff
@@ -168,4 +184,21 @@ resource "equinix_fabric_connection" "secondary_cloud_router_connection" {
       }
     }
   }
+
+  dynamic "z_side" {
+    for_each = var.zside_ap_type == "VD" ? [1] : []
+    content {
+      access_point {
+        type = "VD"
+        virtual_device {
+          type = var.zside_vd_type
+          uuid = var.zside_vd_sec_uuid
```

Comment:
This should still be the same vd_uuid as the first connection. The module implies 2 Cloud Routers to 1 VD. If we have the virtual-device redundant connection then we can have 2 VD's to 1 Cloud Router. We won't have one module that does 2-2.

Reply:
The FCR-2NE-Redundant example supports using two virtual devices (Primary and secondary with different UUIDs), so we will have to add variables for the secondary virtual device